content/id/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -0,0 +1,459 @@
+Perlu dicatat bahwa Horizontal Pod Autoscaling tidak dapat diterapkan pada objek yang tidak dapat di-*scale*, seperti DeamonSets. 
+
+Horizontal Pod Autoscaler diimplementasikan sebagai Kubernetes API *resource* dan sebuah kontroller.
+*Resource* tersebut akan menentukan perilaku dari kontrolernya.
+Kontroler akan menyesuaikan jumlah replika pada kontroler replikasi atau pada *deployment* untuk menyesuaikan dengan hasil observasi rata-rata
+penggunaan CPU sesuai dengan yang ditentukan oleh pengguna.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Bagaimana cara kerja Horizontal Pod Autoscaler?
+
+![Horizontal Pod Autoscaler diagram](/images/docs/horizontal-pod-autoscaler.svg)
+
+Horizontal Pod Autoscaler diimplementasikan sebagai sebuah *control loop*, yang secara
+berkala dikontrol oleh *flag* `--horizontal-pod-autoscaler-sync-period` pada kontroler manajer
+(dengan nilai standar yaitu 15 detik). 
+
+Dalam setiap periode, kontroler manajer melakukan kueri penggunaan sumber daya dan membandingkan
+dengan metrik yang dispesifikasikan pada HorizontalPodAutoscaler. Kontroler manajer mendapat
+metrik dari *resource* metrik API (untuk metrik per *pod*) atau dari *custom* metrik API (untuk semua metrik lainnya).
+
+During each period, the controller manager queries the resource utilization against the
+metrics specified in each HorizontalPodAutoscaler definition.  The controller manager
+obtains the metrics from either the resource metrics API (for per-pod resource metrics),
+or the custom metrics API (for all other metrics).
+
+* Untuk metrik per *pod* (seperti CPU), kontroler mengambil metrik dari *resource* metrik API
+  untuk setiap *pod* yang ditargetkan oleh HorizontalPodAutoscaler. Kemudian, jika nilai target penggunaan ditentukan,
+  maka kontroler akan menghitung nilai penggunaan sebagai persentasi dari pengguaan *resource* dari kontainer 
+  pada masing-masing *pod*. Jika target *raw value* ditentukan, maka nilai *raw metric* akan digunakan
+  secara langsung. Kontroller kemudian mengambil nilai rata-rata penggunaan atau *raw values* (tergantung
+  dengan tipe target yang ditentukan) dari semua *pod* yang targetkan dan menghasilkan perbandingan yang
+  digunakan untuk menentukan jumlah replika yang akan di-*scale*.
+
+  Perlu dicatat bahwa jika beberapa kontainer pada pod tidak memiliki nilai *resource request*, penggunaan CPU
+  pada pod tersebut tidak akan ditentukan dan *autoscaler* tidak akan melakukan tindakan apapun untuk metrik tersebut.
+  Perhatikan pada bagian [detail algoritma](#algorithm-details) dibawah ini untuk informasi lebih lanjut mengenai
+  cara kerja algorihma *autoscale*.
+
+* Untuk *custom* metrik per *pod*, kontroler bekerja sama seperti *resource* metrik per *pod*,
+  kecuali *pod* bekerja dengan *raw values*, bukan dengan *utilization values*.
+
+* Untuk objek metrik dan eksternal metrik, sebuah metrik diambil, dimana metrik tersebut menggambarkan
```

Comment:
in Indonesian, the order is opposite ("object metrics" translates to "metrics object")
```suggestion
* Untuk metrik objek dan metrik eksternal, sebuah metrik diambil, dimana metrik tersebut menggambarkan
```